docs/examples/dynamic_provisioning_external_gluster/README.md

```diff
@@ -0,0 +1,572 @@
+plenty of space.  See below for a quick start guide or [installing GluserFS Guide](https://www.gluster.org/)
+
+
+### Environment and prerequisites for this example.
+
+- GlusterFS Cluster running CentOS 7 (2 nodes, each with at least 2 X 200GB raw devices)
+   - gluster25.rhs (192.168.1.205)
+   - gluster26.rhs (192.168.1.206)
+
+- Heketi Server/Client Node running CentOS 7 (Heketi can be installed on one of the Gluster Nodes if needed)
+   - glusterclient3.rhs (192.168.1.203)
+
+- Kubernetes Node running CentOS 7 (1.5 or later, this example will use the latest source from HEAD for 1.6 version)
+   - k8dev3.rhs (192.168.1.209)
+
+
+### Installing and configuring GlusterFS Nodes
+
+Install 2 nodes to run as the GlusterFS storage cluster and simulate an existing legacy environment. 
+This example uses 2 CentOS 7 systems all running latest GlusterFS. The following steps represent a quick guide on
+how to install and configure these nodes to create a storage pool. More advanced and additional information can be found on the official [GlusterFS Site](https://www.gluster.org/)
+
+***NOTE:*** 2 node clusters are not officially recommended as it causes some known replication issues (split brain), and if possible a 3+ node cluster should be used if possible.
+
+#### View basic node and device information.
+
+Execute lsblk command to show our raw devices (which will be used later in this example to load into Heketi).
+
+```
+	# lsblk
+	NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+	sda           8:0    0   20G  0 disk 
+	├─sda1        8:1    0    1G  0 part /boot
+	└─sda2        8:2    0   19G  0 part 
+	  ├─cl-root 253:0    0   17G  0 lvm  /
+	  └─cl-swap 253:1    0    2G  0 lvm  [SWAP]
+	sdb           8:16   0  200G  0 disk  <1>
+	sdc           8:32   0  200G  0 disk  <2>
+	sr0          11:0    1 1024M  0 rom  
+```
+<1> 200GB raw device that we will let Heketi manage
+
+<2> 200GB raw device that we will let Heketi manage
+
+
+#### Confirm proper access and communication between the storage nodes.
+
+- Update /etc/hosts with ip and hostname (i.e.  192.168.1.205 gluster25.rhs)
+- Set up password-less SSH communication between all nodes
+
+```
+	# ssh-keygen  (take defaults)
+	# ssh-copy-id -i /root/.ssh/id_rsa.pub root@gluster25.rhs
+```
+***NOTE:*** Repeat this from each node as necessary
+
+
+#### Download and install GlusterFS.
+
+
+```
+	# yum install wget telnet -y
+	# yum install centos-release-gluster -y
+	# yum install epel-release -y
+	# yum install glusterfs-server -y
+```
+
+#### Start glusterd.
+
+```
+	# systemctl start glusterd
+	# systemctl enable glusterd
+```
+
+#### Enable firewall rules.
+
+Add Some Firewall Rules On All Nodes (if you have not disabled firewalld)
+
+```
+	# firewall-cmd --zone=public --add-service=glusterfs --permanent
+	# firewall-cmd --reload
+```
+
+
+#### Setup the Storage Pool.
+
+From one of the GlusterFS storage nodes, peer probe the other members of the pool
```

Comment:
agreed, but this is supposed to represent an already functioning glusterfs cluster "brownfield" so that is why I kept it here.  I can add a note to let them know that Heketi can also perform this task?